arch/arm/core/fault.c

```diff
@@ -490,11 +540,87 @@ static void _FaultDump(const NANO_ESF *esf, int fault)
  * @param esf ESF on the stack, either MSP or PSP depending at what processor
  *            state the exception was taken.
  */
+#if defined(CONFIG_ARM_SECURE_FIRMWARE)
+void _Fault(const NANO_ESF *esf, u32_t exc_return)
```

Comment:
can we have the same function signature in both cases and just mark exc_return usused in the CONFIG_ARM_SECURE_FIRMWARE case.

Reply:
Alright, I am going to modify this accordingly.